updating/updating-cluster-between-minor.adoc

```diff
@@ -9,7 +9,7 @@ You can update, or upgrade, an {product-title} cluster between minor versions.
 
 [NOTE]
 ====
-Because of the difficulty of changing update channels by using `oc`, use the web console to change the update channel. It is recommended to complete the update process within the web console. You can follow the steps in xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] to complete the update after you change to a {product-version} channel.
+Use the web console or `oc adm upgrade channel ...` to change the update channel. You can follow the steps in xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] to complete the update after you change to a {product-version} channel.
```

Comment:
```suggestion
Use the web console or `oc adm upgrade channel` to change the update channel. You can follow the steps in xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] to complete the update after you change to a {product-version} channel.
```
Unless they would literally type `...`. 
If `...` is meant to stand in for a parameter, it could be included like this:
> Use the web console or `oc adm upgrade channel _<parameter_name_or_type>_` to change the update channel.

Reply:
moved to `_<channel>_` with 4085e6dff -> 4244510df

Reply:
And just confirming, the `_` give us italics despite being inside backticks:

![image](https://user-images.githubusercontent.com/209920/127202832-178e76e9-97fe-4a7e-9e41-71ddc9647e19.png)

Reply:
yep, italics and brackets denote user-replaced content :) Thanks!